### PR TITLE
Add minimum completion test for subsection visibility

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3921](https://openedx.atlassian.net/browse/DOC-3921)

With WL-1239, course staff can configure prerequisite course subsections based on a minimum score (as exists now) AND OR a percentage completion. This enables pre-assessments that must be completed, but don't require any level of success.

### Date Needed (optional)

This feature is currently in code review.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @ziafazal 
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review: @jshin-edx 
- [ ] PM review: 

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Post-review

- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/developing_course/controlling_content_visibility.rst [hunk 1]
```diff
@@ -260,15 +260,36 @@ earned a minimum score in a prerequisite subsection, follow these steps.
...
    ``75`` to require at least three correct answers.
 
+   To require that learners complete a minimum portion of the prerequisite
+   section, enter the percent of the prerequisite section that learners must
+   complete in the **Minimum Completion Percentage** field. A learner must
+   complete at least this percentage of the prerequisite subsection in order
+   to satisfy the prerequisite and display the current subsection.
+
+   If you set both the **Minimum Score** field and the **Minimum Completion
+   Percentage** field to a value greater than zero, then learners must satify
```
Comment (ziafazal) [inline en_us/shared/developing_course/controlling_content_visibility.rst]: typo satify

**Review by ziafazal — APPROVED** [review]
@grantgoodmanedX one typo other than that looks good.

File: en_us/shared/developing_course/controlling_content_visibility.rst [hunk 2]
```diff
@@ -225,7 +225,7 @@ Create a Prerequisite Subsection
 ==================================
 
 To prevent learners from seeing a subsection of your course until they have
-earned a minimum score in a prerequisite subsection, follow these steps.
+earned a minimum score or completed a minimum portion in a prerequisite subsection, follow these steps.
```
Comment (srpearce) [inline en_us/shared/developing_course/controlling_content_visibility.rst]: Line wrap

File: en_us/shared/developing_course/controlling_content_visibility.rst [hunk 3]
```diff
@@ -260,15 +260,36 @@ earned a minimum score in a prerequisite subsection, follow these steps.
...
+           Prerequisite, Minimum Score, and Minimum Completion Percentage
+           controls.
+
+#. To require that learners achieve a minimum score in the prerequisite
+   subsection, enter the percent of the total score that learners must earn in
+   the **Minimum Score** field. A learner's score for all problems in the
    prerequisite subsection must be equal to or greater than this percentage in
-   order to satisfy the prerequisite and display the current subsection.
+   order to satisfy the prerequisite and display the current subsection. If
+   the prerequisite section is not a problem set, set the **Minimum Score** to
```
Comment (srpearce) [inline en_us/shared/developing_course/controlling_content_visibility.rst]: either "set **Minimum Score** to" or "set the **Minimum Score** field to"

File: en_us/shared/developing_course/controlling_content_visibility.rst [hunk 4]
```diff
@@ -260,15 +260,36 @@ earned a minimum score in a prerequisite subsection, follow these steps.
...
 
    For example, if the prerequisite subsection includes four problems and each
    problem is worth the same number of points, set the **Minimum Score** to
    ``75`` to require at least three correct answers.
 
+   To require that learners complete a minimum portion of the prerequisite
+   section, enter the percent of the prerequisite section that learners must
+   complete in the **Minimum Completion Percentage** field. A learner must
+   complete at least this percentage of the prerequisite subsection in order
+   to satisfy the prerequisite and display the current subsection.
```
Comment (srpearce) [inline en_us/shared/developing_course/controlling_content_visibility.rst]: strike "in order"

Additionally, this feels a little redundant—maybe this:

To require that learners complete a minimum portion of the prerequisite section (also, should this be subsection?) before the current subsection opens, enter the percent of the prerequisite section that learners must complete in the **Minimum Completion Percentage** field.

File: en_us/shared/developing_course/controlling_content_visibility.rst [hunk 5]
```diff
@@ -260,15 +260,36 @@ earned a minimum score in a prerequisite subsection, follow these steps.
...
+       :alt: The Limit Access section in the Advanced settings, showing the
+           Prerequisite, Minimum Score, and Minimum Completion Percentage
+           controls.
+
+#. To require that learners achieve a minimum score in the prerequisite
+   subsection, enter the percent of the total score that learners must earn in
+   the **Minimum Score** field. A learner's score for all problems in the
    prerequisite subsection must be equal to or greater than this percentage in
-   order to satisfy the prerequisite and display the current subsection.
+   order to satisfy the prerequisite and display the current subsection. If
```
Comment (srpearce) [inline en_us/shared/developing_course/controlling_content_visibility.rst]: Strike "in order"

Also, this feels a little redundant. Maybe this:

To require that learners achieve a minimum score of the prerequisite subsection before the current subsection opens, enter the percent of the total score that learners must earn in the **Minimum Completion Percentage** field.

File: en_us/shared/developing_course/controlling_content_visibility.rst (hunk 1 above)
Comment (srpearce) [inline en_us/shared/developing_course/controlling_content_visibility.rst]: Strike "then" (we usually don't include the "then" part of "if... then" statements)

**Review by srpearce — COMMENTED** [review]
👍 with a few minor suggestions

**Pull request merged**